contrib/devtools/symbol-check.py

```diff
@@ -68,6 +68,7 @@
 b'libxcb.so.1', # part of X11
 b'libfontconfig.so.1', # font support
 b'libfreetype.so.6', # font parsing
+b'libdbus-1.so.3', # inter process communication (notification system)
```

Comment:
@theuni: Is this a problem? I guess Qt links to `libdbus` dynamic anyways? And we disable the static library (https://github.com/bitcoin/bitcoin/blob/master/depends/packages/dbus.mk#L9).